corgi/collectors/errata_tool.py

```diff
@@ -225,3 +233,30 @@ def get_builds_for_errata(self, errata_id: int) -> list[int]:
             for component in variant:
                 build_ids.update(component.keys())
         return list(build_ids)
+
+    def _parse_module(
+        self,
+        module_name: str,
+        module_data: dict[str, dict[str, list[str]]],
+        brew: Brew,
+        variant_to_component_map: typing.DefaultDict[str, list],
+    ):
+        """Persist collector models for modular builds from errata build_list.json. This allows
+        later processing of modular builds in the Brew task slow_fetch_modular_build"""
+        for variant, components_by_arch in module_data.items():
+            modular_rpms = []
+            for arch, components in components_by_arch.items():
+                # SRPMS are looked up in persist_modules from Brew via RPMs
+                if arch == "SRPMS":
+                    continue
+                # persist_modules expects the .rpm suffix stripped
+                stripped_component = [
+                    rpm[: -len(".rpm")] for rpm in components if rpm.endswith(".rpm")
+                ]
```

Comment:
We always know the length of ".rpm", so we can hardcode it instead of calling len() on each loop iteration:
```suggestion
                stripped_components = tuple(
                    rpm[:-4] for rpm in components if rpm.endswith(".rpm")
                )
```

Reply:
ok.